cts/cts-cli.in

```diff
@@ -386,7 +386,7 @@ function test_tools() {
     test_assert $CRM_EX_OK
 
     desc="Try to move a resource to its existing location"
-    cmd="crm_resource -r dummy --move --host node1"
+    cmd="crm_resource -r dummy --move --host-uname node1"
```

Comment:
I just realized ... --host-uname is deprecated anyway. We should be using --node.

Reply:
Done.